docs/sources/administration/change-home-dashboard.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Change the default home dashboard"
+description = "How to replace the default home dashboard"
+keywords = ["grafana", "configuration", "documentation", "home"]
+type = "docs"
+[menu.docs]
+name = "Home Dashboard"
+identifier = "change-home-dashboard"
+parent = "admin"
+weight = 100
++++
+
+# Change the default home dashboard
+
+You can change the default homepage of Grafana in two different ways: change Grafana preferences or change the Grafana configuration.
+
+## Change home dashboard preference
+
+You can choose any of your existing Grafana dashboards to replace the default home dashboard in the preferences.
+
+Only administrators can change organization preferences and they affect all the users in the organization. Changing your own user preferences only affects you.
+User preferences override organization preferences. 
+
```

Comment:
I recommend making this two subtasks, each starting with an h3.

### Set home dashboard for your organization
This procedure changes the home dashboard for all users in your organization. Individual user preferences override it.

### Set your personal home dashboard
This procedure only changes your home dashboard. It overrides organization preferences.